source/speech/__init__.py

```diff
@@ -1280,10 +1276,12 @@ def isControlEndFieldCommand(x):
 			if onlyInitialFields or any(isinstance(x, str) for x in speechSequence):
 				yield speechSequence
 			if not onlyInitialFields:
-				yield getSpellingSpeech(

```

Comment:
This might have been a source of generators in speech. It certainly wasn't consistent with the return type defined on the function.